week6/homework/Users.cs

```diff
@@ -0,0 +1,55 @@
+
+using Dapper;
+using MySql.Data.MySqlClient;
+
+namespace HackYourFuture.Week6
+{
+    public interface IUserRepository
+    {
+        Task<IEnumerable<User>> GetUsers();
+        Task<User> PostUser(User userPost);
+        Task<User> PutUser(User user);
+        Task<int> DeleteUser(int id);
+    }
+
+    public class UserRepository : IUserRepository
+    {
+        private string connectionString;
+
+        public UserRepository(IConfiguration configuration)
+        {
+            this.connectionString = configuration.GetConnectionString("Default");
+        }
+
+        public async Task<IEnumerable<User>> GetUsers()
+        {
+            using var connection = new MySqlConnection(connectionString);
+            var users = await connection.QueryAsync<User>("SELECT id,name,phone FROM users1");
+            return users;
+        }
+
+        public async Task<User> PostUser(User user)
+        {
+            using var connection = new MySqlConnection(connectionString);
+            var users = await connection.ExecuteAsync("INSERT INTO dapper.users1 (name, phone) VALUES (@name, @phone)", user);
+            return user;
+        }
+
+        public async Task<User> PutUser(User user)
```

Comment:
A small suggestion here, a more appropriate name for this method would be `Update`